src/lib/common/globals.h

```diff
@@ -82,6 +85,34 @@
 #define DATE_MODIFIED  "dateModified"
 
 
+
+/* ****************************************************************************
+*
+* Render modes - 
+*/
+#define RENDER_MODE_NORMALIZED    "normalized"
+#define RENDER_MODE_KEY_VALUES    "keyValues"
+#define RENDER_MODE_VALUES        "values"
+#define RENDER_MODE_UNIQUE_VALUES "unique"
+
+
+
+/* ****************************************************************************
+*
+* Values for the URI param 'options'
+*/
+#define OPT_COUNT           "count"
+#define OPT_TEXT            "text"
```

Comment:
"text" is no longer an options... Maybe could you use this PR for the little change of removing it?